packages/iTwinUI-react/src/core/Table/actionHandlers/filterHandler.ts

```diff
@@ -10,21 +10,15 @@ export const onFilterHandler = <T extends Record<string, unknown>>(
   action: ActionType,
```

Comment:
I believe you can remove not used props.

Reply:
These props are now being used since the filter checking is done in `filterHandler.ts`